package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oref0",
-  "version": "0.7.0",
+  "version": "0.7.1",
```

Comment:
I originally based this branch on master, but should I rebase it on dev to avoid this difference?

Reply:
Yes.